package.json

```diff
@@ -27,7 +27,8 @@
   "module": "./dist/index.mjs",
   "types": "./dist/types/index.d.ts",
   "files": [
-    "dist"
+    "dist",
+    "src/css/"
```

Comment:
Originally how we had it before v2 update https://github.com/MetaMask/design-tokens/pull/601/files#r1476736756